packages/@okta/vuepress-site/guides/sign-into-mobile-app/section02/index.md

```diff
@@ -13,9 +14,10 @@ In Okta, applications are OpenID Connect (OIDC) clients that can use Okta Author
 7. On the **General** tab of the app that you just created, click **Edit** and enter the correct Okta scheme in the **Logout redirect URIs** box, for example, `com.oktaorg.name:/logout`.
 
 ## Things You Need
+
 After you create the application, there are two values that you need:
 
-**Client ID** - Find it in the applications list or on the **General** tab of a specific application.
+**Client ID** - Find it in the applications list or on the **General** tab of a specific application.<br>
```

Comment:
there are a few places like this in the guides - We're putting things on multiple lines with <br>, which is generally considered a bad practice for the web.  Most of the reasons it's bad don't apply to guides, but it's still worth asking if making these a list would make more sense.